### ISSUE TITLE
operator: log spew about connecting to ceph cluster

### ISSUE THREAD

**Issue by jbw976** [rook/rook#659]
Every 10 seconds, we get log messages about connecting to the ceph cluster.  Either those messages should be at a different level, or we should look into what we're doing every 10s to cause them:
```
2017-05-05 19:07:04.474601 I | cephmon: connecting to ceph cluster rook with user admin
2017-05-05 19:07:04.480110 I | cephmon: successfully connected to cluster rook with user admin
2017-05-05 19:07:14.488063 I | cephmon: connecting to ceph cluster rook with user admin
2017-05-05 19:07:14.493674 I | cephmon: successfully connected to cluster rook with user admin
2017-05-05 19:07:24.502009 I | cephmon: connecting to ceph cluster rook with user admin
2017-05-05 19:07:24.507540 I | cephmon: successfully connected to cluster rook with user admin
```

If we change the logging level, it may be wise to do it as an argument to the connect function.  I have a feeling there are some callers where we really would want it logged at the default INFO level.

**Comment by travisn** [comment]
yeah, this has been annoying. these calls are pretty deep down the stack which is why i hadn't fixed it yet. Agreed that we wouldn't want to change to debug level for many places since it's very useful there.

**Comment by jbw976** [comment]
This is fixed now with https://github.com/rook/rook/commit/ec181c039087f93272b3ba2b788fcd7b84e5a162 right @travisn?

**Comment by travisn** [comment]
Yes, fixed with #878
